Components/Queues/QueueConsumer.php

```diff
@@ -37,19 +37,19 @@ protected function initialize(EndpointInterface $endpoint)
     /**
      * @param EndpointInterface $endpoint
      *
-     * @return \Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\QueueDriverInterface
+     * @return QueueDriverInterface
      */
-    protected function getQueueDriver(EndpointInterface $endpoint)
+    protected function getQueueDriver(EndpointInterface $endpoint): QueueDriverInterface
```

Comment:
In this case the expect driver should be an instance of `SyncQueueDriverInterface`

Reply:
Done.